index.bs

```diff
@@ -1211,7 +1211,7 @@ spec: secure-contexts; urlPrefix: https://www.w3.org/TR/secure-contexts/
   <section>
     <h3 id='example1'>Query playback capabilities with {{decodingInfo()}}</h3>
```

Comment:
shucks, I missed another warning.
WARNING: Multiple elements have the same ID 'example1'.
Deduping, but this ID may not be stable across revisions.
https://api.csswg.org/bikeshed/

Can you change the id for the encoding example (l.1251 - gh won't let me comment there) to be 'example2'.

Reply:
Sorry about this, I really should get bikeshed set up locally. Now fixed.